tests/js/client/shell/shell-admin-log.js

```diff
@@ -32,7 +32,19 @@ function adminLogSuite() {
     arango.POST("/_admin/execute", `for (let i = 0; i < 50; ++i) require('console')._log('general=${level}', 'testi');`);
   };
 
+  let oldLogLevel;
+
   return {
+    setUpAll : function() {
+      // save previous log level for "general" topic;
+      oldLogLevel = arango.PUT("/_admin/log/level", { general: "info" }).general;
```

Comment:
`oldLogLevel` must be set to the old value, not the new one that is returned by PUT. Otherwise, after this test, the general log level remains verbose for the remainder of the test run.

```suggestion
      oldLogLevel = arango.GET("/_admin/log/level").general;
      arango.PUT("/_admin/log/level", { general: "info" });
```